extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -9,9 +9,9 @@ define('ext.wikia.adEngine.lookup.lookupFactory', [
 	'use strict';
 
 	function create(module) {
-		var called = false,
-			onResponseCallbacks = [],
-			response = false,
+		var called,
+			onResponseCallbacks,
```

Comment:
Are you sure it's safe? I believe that bidders are called before AdEngineRunner so it will throw an error when someone add responseListener before AdEngineRunner::delayRun call. What do you think about leaving initial state?